evm_loader/lib/src/types/mod.rs

```diff
@@ -9,26 +20,84 @@ use evm_loader::{
         vector::VectorVecExt, vector::VectorVecSlowExt, AccessListTx, LegacyTx, TransactionPayload,
     },
 };
+use serde::{Deserialize, Serialize};
 use serde_with::skip_serializing_none;
+use serde_with::{hex::Hex, serde_as, DisplayFromStr, OneOrMany};
+use solana_sdk::signature::Signature;
 use solana_sdk::{account::Account, pubkey::Pubkey};
 use std::collections::HashMap;
-pub use tracer_ch_db::ClickHouseDb as TracerDb;
-
-use crate::tracing::TraceCallConfig;
-
-use ethnum::U256;
-use serde::{Deserialize, Serialize};
-use serde_with::{hex::Hex, serde_as, DisplayFromStr, OneOrMany};
-
-use crate::commands::get_config::ChainInfo;
+pub type DbResult<T> = Result<T, anyhow::Error>;
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone, Default)]
+pub struct DbConfig {
```

Comment:
This config value looks strange. The main question is why do you use optional values to different cases? In my opinion, enum with different parameters will be better.